htmlmin/command.py

```diff
@@ -152,8 +150,7 @@ def main():
   if args.input_file:
     inp = codecs.open(args.input_file, encoding=args.encoding)
   else:
-    inp = codecs.getreader(
-      sys.stdin.encoding or locale.getpreferredencoding())(sys.stdin)
+    inp = io.open(0, encoding=args.encoding)
```

Comment:
@mankyd I've updated the code with what I think will work (use stdin/stdout encoding if nothing is specified and otherwise use what's specified or 'utf-8').

But then the arg help text isn't really true (saying it defaults to utf-8).  Thoughts?